docs/integrations/sources/chargebee.md

```diff
@@ -81,6 +81,7 @@ Log into Chargebee and then generate an [API Key](https://apidocs.chargebee.com/
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.6 | XXX | [XXX](https://github.com/airbytehq/airbyte/pull/XXX) | Add support for Event stream |
```

Comment:
can you update here? @joelluijmes

Reply:
Sure, what would you have on the date? For now I'll add tomorrow (if you have then time to test it), but wondering in general. or should it be done when its actually merged?